pkg/controller/autoscaling/elasticsearch/autoscaler/linear_scaler.go

```diff
@@ -68,8 +68,8 @@ func memoryFromStorage(requiredStorageCapacity resource.Quantity, storageRange,
 	requiredMemoryCapacity := memoryRange.Min.Value() + requiredAdditionalMemoryCapacity
 
 	// Round up memory to the next GB
```

Comment:
Nit: unrelated to this commit
```suggestion
	// Round up memory to the next GiB
```